pyproject.toml

```diff
@@ -49,6 +50,9 @@ azure = [
     "azure-mgmt-storage ~= 20.0.0",
     "azure-storage-blob ~= 12.11.0",
     "azure-storage-file-share ~= 12.4.0",
+    "azure-keyvault-secrets ~= 4.7.0", 
+    "azure-keyvault-certificates ~= 4.7.0",
+    "azure-mgmt-authorization ~= 4.0.0",
```

Comment:
is it used?

Reply:
yes, authorization is not, I will delete it